ouroboros-network/src/Ouroboros/Network/MockChain/Chain.hs

```diff
@@ -3,11 +3,10 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE StandaloneDeriving  #-}
 {-# LANGUAGE TypeApplications    #-}
-{-# LANGUAGE PatternSynonyms     #-}
 
 -- | Reference implementation of a representation of a block chain
 --
-module Ouroboros.Network.Chain (
+module Ouroboros.Network.MockChain.Chain (
```

Comment:
@mrBliss is this module only used in `ouroboros-consensus` tests, or is it also required in for `bft` (and other implementations using mocked chains)?  If it's the former, we could move it to `ouroboros-network-testing` package and not confuse others with it.

Reply:
> or is it also required in for `bft` (and other implementations using mocked chains)?

I'm not sure what you mean with this. The BFT protocol implementation? Then the answer is no.

The module in question will be used by the consensus testsuites (both `test-consensus` and `test-storage`), but I believe it will still be mentioned in and imported by parts of the network (e.g., `Ouroboros.Network.Protocol.BlockFetch.Test`) and consensus libraries.

So I don't think we can move it to `ouroboros-network-testing` without additional changes.

Reply:
My point is that if it's only used in test suites of `ouroboros-network` and `ouroboros-consensus` then we can easily move it to `ouroboros-network-testing`.  This can only be done if `ouroboros-consensus` library is not depending on it; I don't think `ouroboros-network` library depends on it.

Reply:
Right, but as I said, both `ouroboros-consensus` and `ouroboros-network` depend on it (even after this PR). For example, `Ouroboros.Network.Protocol.BlockFetch.Test`.